File: app/src/main/java/com/pawel14157/deltadroid/MainActivity.java
package com.pawel14157.deltadroid;

import android.content.Context;
import android.content.Intent;
import android.graphics.Canvas;
import android.graphics.Color;
import android.graphics.Paint;
import android.graphics.Path;
import android.graphics.PointF;
import android.icu.text.DecimalFormat;
import android.os.Build;
import android.support.annotation.RequiresApi;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.view.inputmethod.InputMethodManager;
import android.widget.LinearLayout;
import android.widget.TabHost;



public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }


    /**open Resolve activity
     *
     * @param view
     */
    public void Resolve(View view) {
        Intent resolve = new Intent(this, Resolve.class);
        startActivity(resolve);
        InputMethodManager inputMethodManager = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
        inputMethodManager.toggleSoftInput(InputMethodManager.SHOW_FORCED, 0);

    }

    /*open Help activity
     *
     * @param view
     */
    public void help(View view) {
        Intent help = new Intent(this, Help.class);
        startActivity(help);
    }
}

File: app/src/main/java/com/pawel14157/deltadroid/activity/MainActivity.kt
package com.pawel14157.deltadroid.activity

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import android.view.inputmethod.InputMethodManager
import com.pawel14157.deltadroid.R


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }


    /**open Resolve activity
     *
     * @param view
     */
    fun Resolve(view: View) {
        val resolve = Intent(this, ResolveActivity::class.java)
        startActivity(resolve)
        val inputMethodManager = getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
        inputMethodManager.toggleSoftInput(InputMethodManager.SHOW_FORCED, 0)

    }

    /*open Help activity
     *
     * @param view
     */
    fun help(view: View) {
        val help = Intent(this, HelpActivity::class.java)
        startActivity(help)
    }
}



